### DISCUSSION TITLE
每次尝试获取配置之前都NTP同步时间是必要的么？

### DISCUSSION THREAD

**Discussion by forhuan** [kkkgo/PaoPaoGateWay#103] [问和答] [answered]
https://github.com/kkkgo/PaoPaoGateWay/blob/d0b147356901c8b52d966007dc183ad907f0f8fb/FILES/usr/bin/ppg.sh#L465

**Comment by kkkgo** [comment]
这个是联网成功之后启动ntpd后台服务，不会阻塞获取配置。
部分代理协议对时间精度有要求。

**Reply by forhuan** [reply]
谢谢，是因为看到日志里有很多ntp的dns解析，大概1分多钟1次，所以才有这个疑问。
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/15072138/8d4736fe-9c34-4af1-b5e2-683119bfa13b)
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/15072138/13639a02-df06-4638-9b6f-ca1dbd8d12ff)

**Reply by kkkgo** [reply, marked as answer]
后续可以优化下请求次数。

**Comment by kkkgo** [comment]
最新版本已优化。https://github.com/kkkgo/PaoPaoGateWay/releases
